src/main/java/com/marklogic/kafka/connect/source/MarkLogicSourceConnector.java

```diff
@@ -0,0 +1,61 @@
+package com.marklogic.kafka.connect.source;
+
+import org.apache.kafka.common.config.ConfigDef;
+import org.apache.kafka.connect.connector.Task;
+import org.apache.kafka.connect.source.SourceConnector;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Entry point into the MarkLogic source connector for Kafka. Main purpose is to determine if the user wishes to use
+ * Bulk Data Services or DMSDK for reading data from MarkLogic.
+ */
+public class MarkLogicSourceConnector extends SourceConnector {
+    protected final Logger logger = LoggerFactory.getLogger(getClass());
+
+    static final String MARKLOGIC_SOURCE_CONNECTOR_VERSION = MarkLogicSourceConnector.class.getPackage().getImplementationVersion();
+
+    private Map<String, String> config;
+
+    @Override
+    public ConfigDef config() {
+        return MarkLogicSourceConfig.CONFIG_DEF;
+    }
+
+    @Override
+    public void start(final Map<String, String> config) {
+        logger.info("Starting MarkLogicSourceConnector");
+        this.config = config;
+    }
+
+    @Override
+    public void stop() {
+        logger.info("Stopping MarkLogicSourceConnector");
+    }
+
+    @Override
+    public Class<? extends Task> taskClass() {
+        Class<? extends Task> clazz = RowBatcherSourceTask.class;
+        LoggerFactory.getLogger(getClass()).info("Task class: " + clazz);
```

Comment:
This logging is only for showing whether DMSDK or Bulk DS is being used by the sink connector; I don't think it adds any value here since we'll only have one source task, so I don't think it's worth logging